staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -224,6 +229,17 @@ func NewAvailableConditionController(
 }
 
 func (c *AvailableConditionController) sync(key string) error {
+	select {
+	case <-c.hasBeenReady: // the server is ready and we can perform the checks
```

Comment:
this will queue events until this is ready, naive question, can we wait to start the controller until the network is ready?

Reply:
running the controllers is important for availability, actually, we should always report success and only skip reporting failures if we haven't become ready. let me update the PR.

Reply:
done, PTAL

Reply:
what's the backoff by default in case of errors?